p2p/rlpx.go

```diff
@@ -126,12 +126,12 @@ func (t *rlpx) doProtoHandshake(our *protoHandshake, peer discover.NodeID) (thei
 	// as the error so it can be tracked elsewhere.
 	// Read their handshake before sending ours, to prevent them from rejecting us early
 	if their, receivedAt, err = readProtocolHandshake(t.rw, peer); err != nil {
-		return nil, nil, err
```

Comment:
Lol why was this nil before? bug?

Reply:
lol well, at the time i didn't care what's returned in those lines as they occur when protocol handshakes fail. Later I decided to log the disconnect due to `TooManyPeers` messages and realized that also falls under failure, so in the end, the received timestamp should be just returned in any case.